src/actions/search/index.js

```diff
@@ -5,18 +5,18 @@ import { SearchQueue } from './utils';
 
 let queue;
 
-export function search(query) {
+export function search(query, field) {
```

Comment:
Use the same argument order for all actions creators, i.e. `func(field, any, other, args)`.

Reply:
Fixed by 9a470f15a85febd6ff8f9c776f3a040a8a2ca45c